d2bs/kolbot/libs/common/Storage.js

```diff
@@ -98,6 +98,7 @@ var Container = function (name, width, height, location) {
 		}
 
 		this.itemList = [];
+		this.openPositions = this.height * this.width;
```

Comment:
Could you remove this as it has no purpose, or im i missing something?

Reply:
Yes, this needs to be here to set the available spots in the buffer back to max.

Storage.CanFit() will end up calling Storage.Reset() where this is and without this change the spots left can become negative.